### PR TITLE
fix(deps):  Update dependencies (Kotlin 1.6)

### PR DESCRIPTION
**Pull request by Filmaluco**
Part of https://github.com/relaycorp/relayverse/issues/39

### CODE REVIEW

**Review by sdsantos — APPROVED** [review]

File: app/lint.xml [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
     <issue id="LockedOrientationActivity" severity="ignore" />
     <issue id="GoogleAppIndexingWarning" severity="ignore" />
     <issue id="Overdraw" severity="ignore" />
     <issue id="SelectableText" severity="ignore" />
     <issue id="UnusedIds" severity="informational" />
     <issue id="UnusedResources" severity="informational" />
+    <issue id="ConvertToWebp" severity="informational" />
+    <issue id="TrustAllX509TrustManager">
+        <ignore path="**/JcaJceUtils*.class" />
+    </issue>
```
Comment (gnarea) [inline app/lint.xml:15]: This may point to an actual issue we have to fix. What was the problem?

File: app/lint.xml [hunk 2]
```diff
@@ -3,15 +3,19 @@
...
     <issue id="InvalidPackage">
         <!-- Ignore errors about BC importing javax.naming because we don't use those modules -->
         <ignore path="**/bcpkix-*.jar" />
+        <ignore path="**/kotlinx-coroutines-core-jvm-1.5.2.jar" />
```
Comment (gnarea) [inline app/lint.xml:19]: Please include the reason.

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -58,12 +58,6 @@
         <item name="android:textAllCaps">false</item>
     </style>
 
-
-    <style name="Widget.Ping.TextInputLayout" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox">
-        <item name="boxStrokeColor">?android:textColorSecondary</item>
-        <item name="hintTextColor">?android:textColorSecondary</item>
-    </style>
```
Comment (gnarea) [inline app/src/main/res/values/styles.xml:65]: Was this reported as unused? It's just to know if this requires testing (esp. on older devices).

**Review by gnarea — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (Filmaluco) [reply app/src/main/res/values/styles.xml:65]: it was reported as unused :)

File: app/build.gradle [hunk 4]
```diff
@@ -115,41 +115,39 @@ dependencies {
...
-    implementation 'com.github.relaycorp:awala-endpoint-android:1.10.1'
+    implementation 'com.github.relaycorp:awala-endpoint-android:1.12.3'
 
     // UI
-    implementation 'dev.chrisbanes.insetter:insetter:0.6.0'
+    implementation 'dev.chrisbanes.insetter:insetter:0.6.1'
 
     // Preferences
     implementation 'androidx.preference:preference-ktx:1.1.1'
     implementation 'com.fredporciuncula:flow-preferences:1.5.0'
```
Comment (Filmaluco) [inline app/build.gradle:138]: needs higher kotlin version

File: app/lint.xml (hunk 1 above)
Comment (gnarea) [reply app/lint.xml]: Right, I thought that it was introduced by a new dependency version!

**Review by gnarea — APPROVED** [review]
Obrigado!

**Pull request merged**